core/templates/dev/head/pages/exploration-editor-page/settings-tab/settings-tab.directive.ts

```diff
@@ -125,12 +130,20 @@ angular.module('oppia').directive('settingsTab', [
           ctrl.isRolesFormOpen = false;
 
           ctrl.TAG_REGEX = constants.TAG_REGEX;
-          ctrl.canDelete = GLOBALS.canDelete;
-          ctrl.canModifyRoles = GLOBALS.canModifyRoles;
-          ctrl.canReleaseOwnership = GLOBALS.canReleaseOwnership;
-          ctrl.canUnpublish = GLOBALS.canUnpublish;
+          ctrl.canDelete = false;
+          ctrl.canModifyRoles = false;
+          ctrl.canReleaseOwnership = false;
+          ctrl.canUnpublish = false;
           ctrl.explorationId = ExplorationDataService.explorationId;
 
+          UserExplorationPermissionsService.getPermissionsAsync()
+            .then(function(permissions) {
+              ctrl.canDelete = permissions.can_delete;
```

Comment:
Do we have any permission object in frontend?

Reply:
The permissions object is the data gotten from the `UserExplorationPermissionsService`

Reply:
Ah, I mean do we have any frontend domain object for permissions if yes then lets handel this with that.

Reply:
No, we don't have any frontend domain object